sdk/analytics/src/main/java/com/klaviyo/analytics/networking/requests/KlaviyoApiRequest.kt

```diff
@@ -282,12 +282,12 @@ internal open class KlaviyoApiRequest(
         }
 
         status = Status.Inflight
+        attempts++
 
         return try {
             val connection = buildUrlConnection()
```

Comment:
Incrementing `attempts` is what updates the header value on this request object. But `buildUrlConnection()` is where we actually write the headers into the request. So incrementing has to come first.